megamek/src/megamek/client/ui/swing/AccessibilityWindow.java

```diff
@@ -165,6 +190,16 @@ private void systemEvent(String s) {
         }
     }
 
+    private String cleanHtml(String str) {
+        str = str.replaceAll(cleanHtmlRegex, "");
+        //replace &nbsp; with space
+        str = str.replace("&nbsp;", " ");
+        //replace &amp; with &
+        str = str.replace("&amp;", "&");
+
+        return str;
+    }
+
```

Comment:
Was this used or just left for debugging help?

Reply:
This is used (See line 75), there are complaints about it begin hard to switch between windows in the application, see for example https://github.com/MegaMek/megamek/issues/2651 and so it was asked if the accessibility window could mirror the log for phase reports... I am taking that report from client.phaseReport and stripping the html from it, it's not perfect, but initial user feedback is that it is easier to use. Ideally I want a cut down and cleaned up report for only the current phase, but I don't know how to get that.

Reply:
I missed that somehow! My apologies.